### PR TITLE
preprocess_refactor

### PR DESCRIPTION
**Pull request by MikeLippincott**
Refactoring the data preprocessing prior to HPC run. 
`H202_100_uM` and `H2O2_100_nM` were merged into one `H2O2_100` treatment

### CODE REVIEW

File: 0.Preprocess_Data/scripts/2.preprocessing_nELISA_data.py
Comment (d33bs) [inline 0.Preprocess_Data/scripts/2.preprocessing_nELISA_data.py:232]: Consider removing these lines if they are no longer needed.

File: 0.Preprocess_Data/scripts/2.preprocessing_nELISA_data.py
Comment (d33bs) [inline 0.Preprocess_Data/scripts/2.preprocessing_nELISA_data.py:326]: Are there reasons for using CSV instead of parquet here? Mostly I ask out of curiosity. There may be a benefit to sticking with one format consistently for any outputs, depending on your constraints.

File: 0.Preprocess_Data/scripts/3.data_aggregation.py
Comment (d33bs) [inline 0.Preprocess_Data/scripts/3.data_aggregation.py:24]: Consider using solely the True boolean value of the variables or the `not` keyword to help with these if checks. Also, I recommend using the `and` keyword instead of `&` unless performing an explicit bitwise AND.

For example:
```suggestion
if aggregation and nomic:
```

File: 0.Preprocess_Data/scripts/3.data_aggregation.py
Comment (d33bs) [inline 0.Preprocess_Data/scripts/3.data_aggregation.py:28]: Similarly to the above:
```suggestion
elif not aggregation and nomic:
```

File: 0.Preprocess_Data/scripts/3.data_aggregation.py
Comment (d33bs) [inline 0.Preprocess_Data/scripts/3.data_aggregation.py:32]: Similarly to the above:
```suggestion
elif aggregation and not nomic:
```

File: 0.Preprocess_Data/scripts/3.data_aggregation.py
Comment (d33bs) [inline 0.Preprocess_Data/scripts/3.data_aggregation.py:66]: If `df_nomic` is `None`, will we be able to continue any operations below? If not, consider using `raise` to prevent further flow via an Exception (instead of relying on the implicit exceptions later).

**Review by d33bs — COMMENTED** [review]
Great efforts! I thought things mostly looked good but there were a few areas I wanted to be sure to get your feedback on before an approval from me - especially with concerns to None-types and 0 conversions. Let me know what you think; I could be very off with my asks here. Please don't hesitate to let me know if you have any questions.

File: 0.Preprocess_Data/scripts/1.preprocessing_morphology_data.py
Comment (MikeLippincott) [reply 0.Preprocess_Data/scripts/1.preprocessing_morphology_data.py:34]: Regex will be by default false soon, so I included it anyways.

File: 0.Preprocess_Data/scripts/1.preprocessing_morphology_data.py
Comment (MikeLippincott) [reply 0.Preprocess_Data/scripts/1.preprocessing_morphology_data.py:212]: yes, this gives the proper result. This column in a string of multiple columns combined and so any row that results in 0 can be dropped

File: 0.Preprocess_Data/scripts/1.preprocessing_morphology_data.py
Comment (MikeLippincott) [reply 0.Preprocess_Data/scripts/1.preprocessing_morphology_data.py:284]: fixed

File: 0.Preprocess_Data/scripts/1.preprocessing_morphology_data.py
Comment (MikeLippincott) [reply 0.Preprocess_Data/scripts/1.preprocessing_morphology_data.py:216]: implemented with the `.round` function

File: 0.Preprocess_Data/scripts/2.preprocessing_nELISA_data.py
Comment (MikeLippincott) [reply 0.Preprocess_Data/scripts/2.preprocessing_nELISA_data.py:71]: implemented!

File: 0.Preprocess_Data/scripts/2.preprocessing_nELISA_data.py
Comment (MikeLippincott) [reply 0.Preprocess_Data/scripts/2.preprocessing_nELISA_data.py:85]: I will probably just replace for now as it is a single step and I want the spaces to be replaces with a `_` delimiter

File: 0.Preprocess_Data/scripts/2.preprocessing_nELISA_data.py
Comment (MikeLippincott) [reply 0.Preprocess_Data/scripts/2.preprocessing_nELISA_data.py:95]: Added comments

File: 0.Preprocess_Data/scripts/2.preprocessing_nELISA_data.py
Comment (MikeLippincott) [reply 0.Preprocess_Data/scripts/2.preprocessing_nELISA_data.py:131]: implemented

File: 0.Preprocess_Data/scripts/2.preprocessing_nELISA_data.py
Comment (MikeLippincott) [reply 0.Preprocess_Data/scripts/2.preprocessing_nELISA_data.py:326]: changed to parquet

File: 0.Preprocess_Data/scripts/3.data_aggregation.py
Comment (MikeLippincott) [reply 0.Preprocess_Data/scripts/3.data_aggregation.py:175]: Yes, added!

**Review by d33bs — APPROVED** [review]
Thank you for responding to the comments, LGTM!

**Pull request merged**
